plugins/grpc-plugin/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+edition = "2021"
+name = "cln-grpc-plugin"
+version = "0.1.0"
```

Comment:
if it is incompleted (request and response missed) maybe we can use the convention like this one?
```suggestion
version = "0.0.1"
```

And release the `0.1.0` only after all req/resp are supported?

Reply:
Good idea :+1: